system-variables.md

```diff
@@ -1707,6 +1715,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 开启该变量后，即可在 TiDB Dashboard 中查看查询的图形化执行计划。注意，TiDB Dashboard 只显示变量开启时产生的查询的执行计划。
 - 用 `select tidb_decode_binary_plan('xxx...')` SQL 语句可以从编码后的执行计划解析出具体的执行计划。
 
+### `tidb_gogc_tuner_threshold`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：否
+- 默认值：`0.6`
+- 范围：`[0, 0.9)`
+- 这个变量用来控制 GOGC Tuner 自动调节的最大内存阈值，超过阈值后 GOGC Tuner 会停止工作。
```

Comment:
```suggestion
- 这个变量用来控制 GOGC Tuner 自动调节的最大内存用量阈值，超过阈值后 GOGC Tuner 会停止工作。
```

Reply:
It is not the usage of memory. it is the threshold of memory whether to tune gogc.